tenets/codelingo/go/unused-private-const-var-same-package/codelingo.yaml

```diff
@@ -0,0 +1,32 @@
+tenets:
+  - name: unused-private-const-var-same-package
+    doc: |
+      Find private const or var which are unused in a given package.
+    flows:
+      codelingo/review:
+        comment: |
+          '{{thingname}}' is private yet it is not referenced anywhere else in its package.
```

Comment:
`thingname`?  😆

Reply:
OK I'll change it. haha.

Reply:
Yeah, you should use a more descriptive var name e.g. {{globalname}}. Also, comment should be "Private global {{globalname}} is never used".